app/code/Magento/Directory/Setup/Patch/Data/AddDataForUkraine.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Directory\Setup\Patch\Data;
+
+use Magento\Directory\Setup\DataInstaller;
+use Magento\Directory\Setup\DataInstallerFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Add Ukraine Regions
+ */
+class AddDataForUkraine implements DataPatchInterface
+{
+    /**
+     * @var ModuleDataSetupInterface
+     */
+    private $moduleDataSetup;
+
+    /**
+     * @var DataInstallerFactory
+     */
+    private $dataInstallerFactory;
+
+    /**
+     * AddDataForUkraine constructor.
+     *
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     * @param DataInstallerFactory $dataInstallerFactory
+     */
+    public function __construct(
+        ModuleDataSetupInterface $moduleDataSetup,
+        DataInstallerFactory $dataInstallerFactory
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->dataInstallerFactory = $dataInstallerFactory;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function apply()
+    {
+        /** @var DataInstaller $dataInstaller */
+        $dataInstaller = $this->dataInstallerFactory->create();
+        $dataInstaller->addCountryRegions(
+            $this->moduleDataSetup->getConnection(),
+            $this->getDataForUkraine()
+        );
+
+        return $this;
+    }
+
+    /**
+     * Ukraine regions data.
+     *
+     * @return array
+     */
+    private function getDataForUkraine()
```

Comment:
Please move the private method at the end of class
```suggestion
    private function getDataForUkraine(): array
```

Reply:
👍 agree with return type